src/content/en/fundamentals/media/manipulating/files.md

```diff
@@ -0,0 +1,329 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: TBD.
+
+{# wf_updated_on: 2017-06-16 #}
+{# wf_published_on: 2017-06-16 #}
+
+# Media File Manipulation {: .page-title }
+
+{% include "web/_shared/contributors/josephmedley.html" %}
+
+Now that I've introduced you to applications that can be used for media file
+manipulation, I'm going to take a raw video file off a camera and transform it
+into an encrypted resource that you can play back using a video library such as
+[Google's Shaka Player](https://shaka-player-demo.appspot.com/demo/). I'm
+specifically going to show you how to format your video for mobile playback.
+
+Objective: This article provides explanations of file manipulation concepts,
+with command lines only to illustrate the concepts. There is a companion [cheat
+sheet](cheatsheet) that shows more commands and is designed as a quick reference
+for someone who knows the concepts.
+
+To be more specific about the goals, the result of the procedures described here
+will be media resources with the following characteristics:
+
++  Audio and video streams are split into separate files
++  Versions of the video file are in mp4 and webm format
++  Versions of the audio file are in m4a and webm format
++  A bitrate of 0.35 Megabits per seconde (Mbs)
++  Resolution of 1920 by 1080
++  Encrypted
++  Viewable on all major browsers using appropriate technologies
+
+By 'appropriate technologies' I mean **DASH** and **HLS**, which are the two
+primary means of providing video in HTML on the major browsers. What those terms
+mean and how to produce for them is getting a little ahead of things. Before you
+can even use those technologies, you need your media files in the correct
+format, which is the focus of this article.
+
+One final note, selection of the file formats, bitrate and resolution are not
+arbitrary. I've selected these values for speedy playback on the mobile web.
+
+If you want to play along at home, you'll need a raw video file off a camera,
+preferably one that contains both audio and video. If you don't have one handy,
+then here's
+<a href="/web/fundamentals/media/videos/glocken.mov" download>ten seconds of an mov file</a>
+that I took of the
+[Rathaus-Glockenspiel](https://en.wikipedia.org/wiki/Rathaus-Glockenspiel)
+in Munich's MarienPlatz. 
+
+## How are media Files put together?
+
+Before I start manipulating media files, I want to talk a bit about how media
+files are put together. I think of them as being like an onion. The file that
+you see in your operating system shell is a _container_, identified by a file
+extension (mp4, webm, etc.). The container houses one or more _streams_ of up to
+four types, specifically audio, video, captions, and data. Although containers
```

Comment:
There are definitely more than four types of streams, and some media files can have 30, 40 or more streams. Sorry I missed this in 'Application Primers' earlier.